packages/ckeditor5-table/src/ui/inserttableview.js

```diff
@@ -98,6 +102,13 @@ export default class InsertTableView extends View {
 
 				click: bind.to( () => {
 					this.fire( 'execute' );
+				} ),
+
+				keydown: bind.to( ( evt ) => {
+					if ( evt.key === 'Enter') {
+						this.fire( 'execute' );
```

Comment:
No sure which piece of logic is responsible for this but clicking a toolbar button followed by enter creates a 2x2 table (should be 1x1, I guess)
![2022-08-02 09 58 54](https://user-images.githubusercontent.com/1099479/182323512-55e18380-88a6-4ee4-bd32-faa4019ac2be.gif)

Reply:
Will be handled in #12256.